pyproject.toml

```diff
@@ -591,4 +591,5 @@ ignore-words-list = """
     valu,
     watn,
     watstr,
-    wirth"""
+    wirth,
+    ndefined"""
```

Comment:
I'd prefer if this list stayed alphabetically ordered.

Reply:
Maybe I'll just open a PR with all this in one commit from my own fork.

Reply:
See #14525